schema_based_on_diagram.sql

```diff
@@ -0,0 +1,59 @@
+CREATE TABLE "medical_his_treatments"(
+    "id_medical_his" BIGINT NOT NULL,
+    "id_treatments" BIGINT NOT NULL
+);
+ALTER TABLE
+    "medical_his_treatments" ADD PRIMARY KEY("id_medical_his");
```

Comment:
- [ ] Please 🙏🏽 correct the PRIMARY KEY, it should use both id columns 👉🏽 `PRIMARY KEY (id_medical_his, id_treatments),` for the correct specification of the row in a join table.

Reply:
It has these name because is a Join Table to define the many to many relations.

Reply:
Thank you!